input/gcamdata/R/zchunk_L201.socioeconomics_INDIA.R

```diff
@@ -0,0 +1,132 @@
+#' module_gcam.india_L201.socioeconomics_INDIA
+#'
+#' Interest rate, population, and GDP for GCAM-INDIA.
+#'
+#' @param command API command to execute
+#' @param ... other optional parameters, depending on command
+#' @return Depends on \code{command}: either a vector of required inputs,
+#' a vector of output names, or (if \code{command} is "MAKE") all
+#' the generated outputs: \code{L201.InterestRate_GCAMINDIA}, \code{L201.Pop_GCAMINDIA}, \code{L201.BaseGDP_GCAMINDIA}, \code{L201.LaborForceFillout_GCAMINDIA}, \code{L201.LaborProductivity_GCAMINDIA}.
+#' @details Interest rate, population, and GDP for GCAM-INDIA.
+#' @importFrom assertthat assert_that
+#' @importFrom dplyr filter mutate select
+#' @importFrom tidyr gather spread
+#' @author Malyan_Ankur_CEEW
+module_gcam.india_L201.socioeconomics_INDIA <- function(command, ...) {
+  if(command == driver.DECLARE_INPUTS) {
+    return(c(FILE = "gcam-india/States_Subregions_Grid",
+             "L100.Pop_thous_state_india",
+             "L100.GDP_mil90usd_state_india",
+             "L102.pcgdp_thous90USD_GCAM3_ctry_Y"))
+  } else if(command == driver.DECLARE_OUTPUTS) {
+    return(c("L201.InterestRate_GCAMINDIA",
```

Comment:
Please have consistent naming for all GCAM-India csv outputs. `_india` is used for the level1 chunk.

Reply:
Sure. Will follow -India for Level 2 and _India for level 1